lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -158,7 +158,8 @@ class ReportUIFeatures {
       this.addButton({
```

Comment:
Is there anything left before enabling this in DT?

Reply:
yea i had to edit this file

verified working with `yarn open-devtools`